cordformation/src/test/resources/net/corda/plugins/DeployTwoNodeCordappWithDocker.gradle

```diff
@@ -29,6 +29,7 @@ task prepareDockerNodes(type: net.corda.plugins.Dockerform, dependsOn: ['jar'])
             address("localhost:10003")
             adminAddress("localhost:10043")
         }
+        sshdPort 22234
```

Comment:
Is there any way that you can test what has been written into the YAML please? I am worried that we have strayed into "undocumented behaviour" territory and so want to keep a close eye on the output :worried:.

You should be able to find the `docker-compose.yml` file relative to `testProjectDir`.

Reply:
Extended one of the unit tests to check if the ports are correctly set